public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -238,12 +239,17 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
         queries.push(this.createQuery(instantTarget, options, start, end));
         activeTargets.push(instantTarget);
       } else {
-        if (target.exemplar) {
+        // It doesn't make sense to query for exemplars in dashboard if only instant is selected
+        if (target.exemplar && !target.instant) {
           const exemplarTarget = cloneDeep(target);
           exemplarTarget.requestId += '_exemplar';
           target.exemplar = false;
           queries.push(this.createQuery(exemplarTarget, options, start, end));
           activeTargets.push(exemplarTarget);
+          this.exemplarErrors.next();
+        }
+        if (target.exemplar && target.instant) {
+          this.exemplarErrors.next("Exemplars doesn't make sense with instant query.");
```

Comment:
Yeah, I agree with @ifrost, I would rephrase to `Exemplars are not available for instant queries.` because there might be a tiny/edge case-y usecase when it makes sense. So I would just say they are not available. Maybe you can check with docs team how would they phrase it.